### DISCUSSION TITLE
Button error....

### DISCUSSION THREAD

**Discussion by FeelsBadMan1** [kiki7000/discord.py-components#6] [Q&A]
Dear kiki7000,
I tried to use te libary but on the docs and on github are 2 different codes. i tried both of these codes. the fist code not needed to have the componentmodules but as i wrote messages the bot just spammed the word "content" and then today i tried the other code that need the component module but then got this error: Object of type Button is not JSON serializable
Is there a workaround or any solution for that?

**Comment by kiki7000** [comment]
+ The example code sends "content" when a message is created, but it is also triggered by its own message so that is why the bot is spamming "content"
+ The button can't be created by normal users. Look at [this](https://github.com/kiki7000/discord.py-components/discussions/4#discussioncomment-712530)
+ I don't know about the error "Object of type Button is not JSON serializable", so can you tell me the code you used?

**Comment by FeelsBadMan1** [comment]
i used the examplecode on this githubpage.

**Reply by kiki7000** [reply]
Update the module and try it again plz!
